index.js

```diff
@@ -11,6 +13,16 @@ const HapiAlive = require('hapi-alive')
 const HapiDevErrors = require('hapi-dev-errors')
 const server = new Hapi.Server()
 
+const loadHealthTemplate = () => {
```

Comment:
This is more a question for my own understanding. What is the difference between the use of the [hapi-alive](https://github.com/idoshamun/hapi-alive) package and the /`health` route it adds, and what we do for all our other routes?

Basically I'm just wondering could we not just implement this ourselves using the same pattern we use for other routes. As I said, this might just be me missing something.

Reply:
I think that the point of the hapi-alive component is to be a quick and easy way to add a /health route to the application that can be 'pinged' to check service availability.

In this PR I have managed to get it to display some useful information on that /health route, but I think that the more information we try and include then the less suitable this component becomes and the more we might as well just create the route ourselves.

I think the amount of information we currently have is probably about as much as we would want to include if we are using this component, and as you can see it was a bit of a pain to inject in the required data.

As we would be concerned about performance of this route it would be an interesting exercise to compare the time taken to show the /health route vs the time it would take to just create the route ourselves without using hapi-alive. I would hope that the hapi-alive component would be able to return the /health page a lot quicker, but it would be interesting to find out.

Reply:
Let's keep it as is for now then, and if we ever get that 5 minutes we can do the test.